src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/Common.ps1

```diff
@@ -59,6 +59,10 @@ Gets the default location for a provider
 #>
 function Get-ProviderLocation($provider, $preferredLocation = "West Central US")
 {
+    if($env:AZURE_NRP_TEST_LOCATION)
```

Comment:
I think you'll want to move this into the ```if((Get-NetworkTestMode) -ne 'Playback')``` block, as if this environment variable is for some reason set when running in playback, all tests that were not recorded with that location will fail.

Reply:
@maddieclayton done